python/cugraph/cugraph/traversal/sssp.py

```diff
@@ -217,7 +217,7 @@ def sssp(
         G, nx_weight_attr="weight", matrix_graph_type=Graph(directed=directed)
     )
 
-    if not G.edgelist.weights:
+    if not G.is_weighted:
```

Comment:
This is a sneaky bug, since as-written it will always be True. This needs to be changed to actually call the method:
```suggestion
    if not G.is_weighted():
```

Reply:
Can you also add a test to verify both weighted and unweighted Graphs are handled correctly?

Reply:
Right. That was a typo.

Reply:
initially I had it call `G.weighted` but then I changed it to `G.is_weighted()` which call `G.weighted`  and I forgot the parentheses. I am fixing that now

Reply:
Can you also add the test with an unweighted graph and verify it catches your typo?

Reply:
Yep. I am about to push it now

Reply:
Done